app/controllers/news_controller.rb

```diff
@@ -38,6 +38,8 @@ def show
 
   private
   def set_section
-    @section = SectionHome.find_by!(slug: params[:section]).section
+    Rollbar.silenced do
```

Comment:
Looking into the rollbar docs we might be better achieving this with an exception level filter (see https://rollbar.com/docs/notifier/rollbar-gem/#advanced-usage).

The issue I have with silencing rollbar is that if there is an exception raised other than a 404 we'll miss it.